packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -161,13 +160,20 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                                 ) : (
                                     <div
                                         className='p2p-my-ads__table-add'
-                                        onClick={() => {
-                                            setAdvert(advert);
-                                            my_ads_store.showQuickAddModal(advert);
-                                        }}
+                                        onClick={
+                                            !general_store.is_barred &&
```

Comment:
Rather than defining the function here, can it be declared above and be called here. This will be easy to understand and will help in expanding the function body in future.

Reply:
Thx for the suggestion! Will do that